mkosi/__init__.py

```diff
@@ -168,6 +170,28 @@ class MkosiException(Exception):
     """Leads to sys.exit"""
 
 
+def dictify(f):
+    def wrapper(*args, **kwargs):
+        return dict(f(*args, **kwargs))
+    return functools.update_wrapper(wrapper, f)
+
+@dictify
+def read_os_release():
+    try:
+        f = open('/etc/os-release')
+    except FileNotFoundError:
+        f = open('usr/lib/os-release')
+
+    for line_number, line in enumerate(f):
+        m = re.match(r'([A-Z][A-Z_0-9]+)=(.*?)\s*$', line)
```

Comment:
This regex is a bit too strict. When I have a comment or empty line it will print that this line is bad, although the `os-release` man page says that's allowed.

Reply:
Of course. I forgot about comments because I mostly tested on real os-release files, but it's still a stupid omission on my part.